api/oneinch/throttler.go

```diff
@@ -0,0 +1,21 @@
+package oneinch
+
+import (
+	"time"
+)
+
+var (
+	lastRequest       time.Time
+	requestsPerSecond float64 = 1
```

Comment:
This would be better fit in the command line arguments so users have more control (maybe I could add this functionality to the SDK directly instead)

Reply:
I'd rather have the implementation to query for the rate limit from the API rather than to ask the user for it. Most users won't have a clue what to enter anyway.